recipes/libcurl/all/conanfile.py

```diff
@@ -231,14 +235,19 @@ def imports(self):
         #     but does not work on OS X 10.11 with SIP)
         # 2. copying dylib's to the build directory (fortunately works on OS X)
         if self.settings.os == "Macos":
-            self.copy("*.dylib*", dst=self._source_subfolder, keep_path=False)
+            copy(self, "*.dylib*", src=self.build_folder, dst=self.source_folder, keep_path=False)
```

Comment:
In continuation of https://github.com/conan-io/conan-center-index/pull/12377#discussion_r963088772